aws/code/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+source activate rapids
+if [[ "$1" == "serve" ]]; then
+    echo -e "@ entrypoint -> launching serving script \n"
+    python serve.py
```

Comment:
I know we talked about this a bit previously, but it's probably worth revisiting whether or not this should be deployed through a WSGI server for the use case(s) we intend.

Reply:
Agreed, seems like we should schedule a deeper dive conversation around the server (including front-loading the FIL model, etc).